packages/ui/src/primitives/tailwind/Input/index.tsx

```diff
@@ -102,7 +102,8 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
       containerClassname
     )
 
-    const containerClass = variant === 'outlined' ? 'bg-theme-surface-main relative w-full' : ' relative w-full'
+    const containerClass =
+      variant === 'outlined' ? 'bg-theme-surface-main relative h-full w-full' : ' relative h-full w-full'
```

Comment:
@CITIZENDOT do you think this will cause ui problems in the multi-tenancy? (am seeing none in studio and admin)

Reply:
AFAIK `h-full` doesn't have any effect here, since parent tag doesn't have a fixed height. [Why parent tag should have fixed height for `h-full` to work?](https://stackoverflow.com/a/7049918/12347371)